### ISSUE TITLE
Upgrade to Chockidar 3

### ISSUE THREAD

**Issue by isaacabraham** [SAFE-Stack/SAFE-template#462]
Building a SAFE v3 project currently gives the following:

```
chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
```

I got stung by this recently. People that have newer versions of node will simply get a crappy error message on first time they use SAFE.

**Comment by theimowski** [comment]
I still can't get my head around npm dependencies - we actually have chokidar pinned to version 3.5.1. I assume that this warning comes from one of the transient dependencies - e.g. `webpack` -> `watchpack` -> `watchpack-chokidar2` -> `chokidar ^2.1.8` 
Not sure if one can mute such warnings given that we already use newer version, or whether the only solution is to update webpack family to version 5

**Comment by isaacabraham** [comment]
Was Chokidar always pinned? Let me test this again with newer Node - maybe I had an older version of the template running.

**Comment by isaacabraham** [comment]
Fixed by #472
